server/plugin/utils.go

```diff
@@ -31,17 +31,28 @@ func (p *Plugin) sendEphemeralPostForCommand(args *model.CommandArgs, text strin
 }
 
 // DM posts a simple Direct Message to the specified user
-func (p *Plugin) DM(mattermostUserID, format string, args ...interface{}) (string, error) {
+func (p *Plugin) DM(mattermostUserID, format string, isSlackAttachment bool, args ...interface{}) (string, error) {
```

Comment:
sync with master and see if there is a need to make changes for wherever `DM` is used

Reply:
I already did the changes wherever `DM` was present while adding this flag.